app/users/serializers.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils.http import urlsafe_base64_decode
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+
+from app.users.utils import (
+    create_reset_email,
+    generate_reset_token,
+    send_email,
+)
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
Why do you have this yet it belongs to another story?
it will cause conflicts later

Reply:
I have this because we need to authenticate the user, for it to be possible to reset their password